### ISSUE TITLE
[Feature]: Disable third part cookies in Webview2

### ISSUE THREAD

**Issue by yadav-shivam** [MicrosoftEdge/WebView2Feedback#4336]
### Describe the feature/enhancement you need

I need a setting to disable or block third-party cookies. This feature is available in the Microsoft Edge browser but we need this in our browser which is based on Webview2. 

### The scenario/use case where you would use this feature

We have a browser based on Webview2 where we need to provide the feature to block the third-party cookies for security purposes. 
Please expose some API for the same. This is available in Chromium and Edge as well.

### How important is this request to you?

Critical. My app's basic functions wouldn't work without it.

### What does your app do? Is there a pending deadline for this request?

My App is a Industrial Browser which is running on a deadline to release it soon in the market.

**Comment by victorhuangwq** [comment]
We already have the tracking prevention API in WebView2. https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2trackingpreventionlevel?view=webview2-winrt-1.0.2210.55

Let me know if this is not what you are asking about, if so, could you take a screenshot of the settings in Microsoft Edge that you are speaking of?

**Comment by yadav-shivam** [comment]
@victorhuangwq Thanks for Quick response, just to confirm what you are suggesting I am attaching a screenshot for Edge settings
Please check and confirm if this can be achieved with the API you suggested ?
![BlockThirdPartyEdge](https://github.com/MicrosoftEdge/WebView2Feedback/assets/22475909/58033f99-b0f7-47e1-8adc-1e384597cae9)

**Comment by yadav-shivam** [comment]
@victorhuangwq No! We need the API to block third party cookies as shown in screenshot above.
